src/util/ChartUtils.js

```diff
@@ -245,16 +245,22 @@ export const getBarPosition = ({ barGap, barCategoryGap, bandSize, sizeList = []
       originalSize >>= 0;
     }
     const size = (maxBarSize === +maxBarSize) ? Math.min(originalSize, maxBarSize) : originalSize;
+    // when maxBarSize is small, each bar should only be divided by realBarGap, not wide empty space
+    let sum = sizeList.reduce((res, entry) => (res + size), 0);
```

Comment:
It seems no `size` in `sizeList`.

Reply:
yes, I made a mistake. should change to 
    
const sum = len * size + (len - 1) * barGap;